reference/display.md

```diff
@@ -175,10 +175,7 @@ void wb_display_detach_camera(WbDeviceTag tag);
 
 **Description**
 
-These functions are used to attach/detach a camera to a display. If a camera is attached to a display, as soon as a new image is generated by the camera it is efficiently copied in the display. This is useful to simulate mirrors for example.
-
-> **note**:
-When a camera is attached to a display it is not possible any more to draw on this display.
+These functions are used to attach/detach a camera to a display. When a camera is attached to a display, the camera images are efficiently copied to the background of the display. This is useful to simulate for example mirrors or head-up display (HUD).
```

Comment:
Just for the note, an HUD is not the central mirror of the car but is mechanism like this one: http://continental-head-up-display.com/wp-content/uploads/2014/07/Head_up_display_technology_experience.jpg